www/apps/book/app/learn/update/page.mdx

```diff
@@ -0,0 +1,110 @@
+
+<Note>
+
+This doesn't apply to the design-system packages, including `@medusajs/ui`, `@medusajs/ui-presets`, and `@medusajs/ui-icons`. These packages are versioned independently. However, you don't need to install and manage them separately in your Medusa application, as they are included in the `@medusajs/admin-sdk`. If you're using them in a standalone project, such as a storefront or custom admin dashboard, refer to [this section in the Medusa UI documentation](!ui!/installation/standalone-project#updating-ui-packages) for update instructions.
+
+</Note>
+
+Medusa updates the version number `major.minor.patch` according to the following rules:
+
+- **patch**: A patch release includes bug fixes and minor improvements. It doesn't include breaking changes. For example, if the current version is `2.0.0`, the next patch release will be `2.0.1`.
+- **minor**: A minor release includes new features, fixes, improvements, and breaking changes. For example, if the current version is `2.0.0`, the next minor release will be `2.1.0`.
+- **major**: A major release includes significant changes to the entire codebase and architecture. For those, the update process will be more elaborate. For example, if the current version is `2.0.0`, the next major release would be `3.0.0`.
+
+---
+
+## Check Installed Version
+
+To check the currently installed version of Medusa in your project, run the following command in your Medusa application:
+
+```bash
+npx medusa -v
+```
+
+This will show you the installed version of Medusa and the [Medusa CLI tool](!resources!/medusa-cli), which should be the same.
+
+---
+
+## Check Latest Version
+
+The documentation shows the current version at the top right of the navigation bar. When a new version is released, you'll find a blue dot on the version number. Clicking it will take you to the [release notes on GitHub](https://github.com/medusajs/medusa/releases).
+
+You can also star the [Medusa repository on GitHub](https://github.com/medusajs/medusa) to receive updates about new releases on your GitHub dashboard. Our team also shares updates on new releases on our social media channels.
+
+---
+
+## Update Medusa Application
+
+Before updating a Medusa application, make sure to check the [release notes](https://github.com/medusajs/medusa/releases) for any breaking changes that require actions from your side.
+
+Then, to update your Medusa application, bump the version of all `@medusajs/*` dependencies in your `package.json`. Then, re-install dependencies:
+
+```bash npm2yarn
+npm run install
+```
+
+This will update all Medusa packages to the latest version.
+
+### Running Migrations
+
+Minor releases may include changes to the database, such as new tables, updates to existing tables, updates after adding links, or data migration scripts.
```

Comment:
todo: Patches can also include changes to the database and we don't deem added migrations a breaking change.